_validate/regenerateTranslations.py

```diff
@@ -23,15 +24,25 @@ def regenerateJsonFile(filePath: str, errorFilePath: str | None) -> None:
 			with open(errorFilePath, "w") as errorFile:
 				errorFile.write(f"Validation Errors:\n{manifest.errors}")
 		return
-
+	changelog = manifest.get("changelog")  # type: ignore[reportUnknownMemberType]
+	if changelog == "None":
+		# The config default is None
+		# which is parsed by configobj as a string not a NoneType
+		changelog = None
```

Comment:
I think this code can be removed as per https://github.com/nvaccess/addon-datastore-validation/pull/52/files#r2512900158
 
```suggestion
```